pystac/extensions/item_assets.py

```diff
@@ -2,242 +2,76 @@
 
 from __future__ import annotations
 
-from copy import deepcopy
+import warnings
 from typing import TYPE_CHECKING, Any, Literal
 
 import pystac
+from pystac.errors import DeprecatedWarning
 from pystac.extensions.base import ExtensionManagementMixin
 from pystac.extensions.hooks import ExtensionHooks
+from pystac.item_assets import ItemAssetDefinition
 from pystac.serialization.identify import STACJSONDescription, STACVersionID
 from pystac.utils import get_required
 
 if TYPE_CHECKING:
-    from pystac.extensions.ext import ItemAssetExt
+    pass
```

Comment:
Just nuke this block since it's a `pass` now?

Reply:
lol yep